upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -618,24 +619,24 @@ func validateDNS(cluster *api.Cluster, cloud fi.Cloud) error {
 	zone := matches[0]
 	dnsName := strings.TrimSuffix(zone.Name(), ".")
 
-	glog.V(2).Infof("Doing DNS lookup to verify NS records for %q", dnsName)
-	ns, err := net.LookupNS(dnsName)
-	if err != nil {
-		return fmt.Errorf("error doing DNS lookup for NS records for %q: %v", dnsName, err)
-	}
+	if os.Getenv("DNS_IGNORE_NS_CHECK") == "" {
```

Comment:
I will be using this env as I am using a private hosted zone. I am unsure if there were other cases when this env var would be used, @justinb? 

Maybe we need to be more explicit being a private hosted zone. We could either identify whether the user has given a private or public using the aws API, or have it explicitly defined in the yaml.